ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentSummary/DocumentSummary.js

```diff
@@ -0,0 +1,55 @@
+import { EItemIcon, ItemIcon, LoanIcon } from '@pages/backoffice';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Statistic } from 'semantic-ui-react';
+
+export default class DocumentSummary extends Component {
+  render() {
+    const { document } = this.props;
+    return (
+      <Statistic.Group widths="five" size="tiny">
```

Comment:
IMHO, it would be really nice to have them clickable to that you can jump down to the section, as in the side navigation. Maybe for the next PR?

Reply:
I thought about it, but this is what the right side menu is for, do you think we need the same functionality here?